lib/pr_checker.js

```diff
@@ -211,6 +252,36 @@ class PRChecker {
     // 3. is not authored by the people opening the PR
     return true;
   }
+
+  checkCommitsAfterReview(commit, review, logger) {
+    const commitIndex = commit.length - 1;
+    const reviewIndex = review.length - 1;
+    const lastCommit = commit[commitIndex].commit;
+    const lastReview = review[reviewIndex];
+
+    const commitDate = lastCommit.committedDate;
+    const reviewDate = lastReview.publishedAt;
+
+    if (commitDate < reviewDate) {
+      logger.info('No new commits after last review');
+    } else {
+      warn();
+    }
+
+    function warn() {
+      const multipleCommits = commit.length - 1;
+      const pullURL = lastReview.url.split('#')[0];
+      const commitURL = `${pullURL}/commits/${lastCommit.oid}`;
+      logger.warn('Changes were pushed since the last review:');
+      if (multipleCommits >= 1) {
+        logger.warn(`see ${pullURL}/commits to see all the commits`);
+        logger.warn(`see ${pullURL}/files to see all the changes`);
```

Comment:
I think we don't need this URL because it shows all the changes instead of the changes added later.